spyder/plugins/maininterpreter/confpage.py

```diff
@@ -158,9 +159,48 @@ def setup_page(self):
         umr_layout.addWidget(umr_namelist_btn)
         umr_group.setLayout(umr_layout)
 
+        # Path anaconda executor
+        conda_group = QGroupBox(_("Path Conda executor"))
+        conda_label = QLabel(
+            _(
+                "Select the conda executor path"
+            )
+        )
+        conda_label.setWordWrap(True)
+
+        conda_layout = QVBoxLayout()
+        conda_layout.addWidget(conda_label)
```

Comment:
```suggestion
        conda_group = QGroupBox(_("Conda executable"))
        conda_layout = QVBoxLayout()
```

Rename QGroupBox to be more meaningful and remove unnecessary label.